frontend/packages/knative-plugin/src/components/overview/RevisionsOverviewList.tsx

```diff
@@ -40,6 +45,30 @@ const RevisionsOverviewListItem: React.FC<RevisionsOverviewListItemProps> = ({
         </div>
         <span className="col-lg-4 col-md-4 col-sm-4 text-right">{getTraffic(name)}</span>
       </div>
+      {deploymentData && (
+        <div className="odc-revision-deployment-list">
+          <div className="row">
+            <div className="col-lg-8 col-md-8 col-sm-8">
+              <ResourceLink
+                kind={deploymentData.kind}
+                name={deploymentData.name}
+                namespace={namespace}
+              />
+            </div>
+            {revision.kind === RevisionModel.kind && (
+              <div className="col-lg-4 col-md-4 col-sm-4">
+                <PodRing
+                  pods={current ? current.pods : []}
+                  obj={revision}
+                  rc={current && current.obj}
+                  resourceKind={RevisionModel}
+                  path="/spec/replicas"
+                />
```

Comment:
you can use `PodStatus` component here and pass number pods through `title` prop.

Reply:
The pod ring doesn't match the designs right now. If you use PodStatus, you can pass innerRadius, outerRadius, size through props to match it with UX designs.